repose-aggregator/tests/performance-test-library/build.gradle

```diff
@@ -0,0 +1,4 @@
+dependencies {
+    compile 'io.gatling:gatling-test-framework:2.2.4'
```

Comment:
Ah. Didn't see this the first time. Is there any reason not to stick this version in the version properties file?

Also, is there a way we can tie this version number to the version of Gatling we are installing using Ansible? I can't think of one, and it comes back to our lack of a Gradle/Ansible bridge, but it's something that would be nice to have.